src/code/gfxprint.c

```diff
@@ -222,21 +222,21 @@ void GfxPrint_PrintCharImpl(GfxPrint* this, u8 c) {
 
         if (this->flag & GFXPRINT_FLAG64) {
             gSPTextureRectangle(this->dlist++, (this->posX + 4) << 1, (this->posY + 4) << 1, (this->posX + 4 + 32) << 1,
-                                (this->posY + 4 + 32) << 1, tile, (u16)(c & 4) * 64, (u16)(c >> 3) * 256, 512, 512);
+                                (this->posY + 4 + 32) << 1, tile, (u16)(c & 4) * 64, (u16)(c >> 3) * 256, (1 << 10) / 2, (1 << 10) / 2);
         } else {
             gSPTextureRectangle(this->dlist++, this->posX + 4, this->posY + 4, this->posX + 4 + 32, this->posY + 4 + 32,
-                                tile, (u16)(c & 4) * 64, (u16)(c >> 3) * 256, 1024, 1024);
+                                tile, (u16)(c & 4) * 64, (u16)(c >> 3) * 256, 1 << 10, 1 << 10);
```

Comment:
Why use `1 << 10` instead of `1024` here?

Reply:
because its correct: http://n64devkit.square7.ch/n64man/gsp/gSPTextureRectangle.htm